src/env.h

```diff
@@ -239,6 +241,7 @@ namespace node {
   V(unknown_string, "<unknown>")                                              \
   V(user_string, "user")                                                      \
   V(username_string, "username")                                              \
+  V(usingicudatadir_string, "usingICUDataDir")                                \
```

Comment:
Ok, I've been wondering about that. Good to know :)
On Apr 20, 2016 7:45 AM, "Ben Noordhuis" <notifications@github.com> wrote:

> In src/env.h
> <https://github.com/nodejs/node/pull/6266#discussion_r60420685>:
>
> > @@ -239,6 +241,7 @@ namespace node {
> >    V(unknown_string, "<unknown>")                                              \
> >    V(user_string, "user")                                                      \
> >    V(username_string, "username")                                              \
> > +  V(usingicudatadir_string, "usingICUDataDir")                                \
>
> If you're only using these strings once, there is not much point in adding
> them to the per-isolate cache.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6266/files/a66c3d54f3d6e738977a237aacfb8663b5543e35#r60420685>
>